### ISSUE TITLE
Web search function does not work

### ISSUE THREAD

**Issue by suzizisu** [ellite/Wallos#221]
I'm sure my machine can access google images. But the search results are still blank.

![3](https://github.com/ellite/Wallos/assets/103314072/dc323203-ad42-49a1-8dd3-af2320c88531)
![4](https://github.com/ellite/Wallos/assets/103314072/a2965afa-91b0-4215-9152-030032012947)

**Comment by ellite** [comment]
Hi, that is strange.
Even if google does not respond, there should be a fallback to brave search.

The same search from my test machine:

```
{
    "imageUrls": [
        "https:\/\/encrypted-tbn0.gstatic.com\/images?q=tbn:ANd9GcTFgV5f7XClZ_-skaQj5mdFppiu6fJmEZWsW2X0qTl8ahQZHnyoew-k6joHSg&s",
        "https:\/\/encrypted-tbn0.gstatic.com\/images?q=tbn:ANd9GcRugXyHKY_W2Ibc1oCKoK8QFYAe7_W99BCp6U63BG57K1gQsHd2H9_esL_9Mxw&s",
        "https:\/\/encrypted-tbn0.gstatic.com\/images?q=tbn:ANd9GcQD3YRG2Z8Pu1GEzhUWBltKxsDKJ8pyB1L-JoJa2mJ7ZDJiCUoRoPbNkXMBnes&s",
        "https:\/\/encrypted-tbn0.gstatic.com\/images?q=tbn:ANd9GcSxgUbLfiLfuXqCO8SWhi0WFnE2rDBJso3ZdOcgspvbSI-lBmrRI5q8CkgBfAY&s",
        "https:\/\/encrypted-tbn0.gstatic.com\/images?q=tbn:ANd9GcQ0S9uApgab6j00I80J1SvqSewlRmlplv2s2cb_NL4X_dsuQg1bMDINit6Ugg&s",
        "https:\/\/encrypted-tbn0.gstatic.com\/images?q=tbn:ANd9GcQ9TqmXNGj6pE-iR8NK2mMbkJUCYjiU1FT1u5o1_wUGD2D1b04RmfXgRBuYM-A&s",
        "https:\/\/encrypted-tbn0.gstatic.com\/images?q=tbn:ANd9GcTGIKKsjWTthsmhTcQ2kY7G8m-Hk8zCOSaKPe28l_mwVZPFBFLYWqukRbGti-E&s",
        "https:\/\/encrypted-tbn0.gstatic.com\/images?q=tbn:ANd9GcSkbsLlGrJemD9iSx9hVxgFmYLFVM-jcSmpMu4kSHh7aCxnPUnCNMedfAb4FA&s",
        "https:\/\/encrypted-tbn0.gstatic.com\/images?q=tbn:ANd9GcRpYnjgxFjMJkqeWQ0DfRsQo2PyTUu4Szaa1v37Jf5TCil-ULmX8GTeuW4q6A&s",
        "https:\/\/encrypted-tbn0.gstatic.com\/images?q=tbn:ANd9GcTkPKv7jRFTyFfJec8flTIh4F8-YH08gCk8ULvWY12Kt6D3ZVX6tY2_yOMQpA&s",
        "https:\/\/encrypted-tbn0.gstatic.com\/images?q=tbn:ANd9GcR-OxhD0vbSBZ_IUK0Mr_jAa23cLwrqu_A71LYujXbn_8__uCJ_sRzV_edD0D8&s",
        "https:\/\/encrypted-tbn0.gstatic.com\/images?q=tbn:ANd9GcTnqg5fBu_9GKI-k9wUAshmecxD__lqkMhDqrCppVARxsXbHDQ6IN09PuP4SYk&s",
        "https:\/\/encrypted-tbn0.gstatic.com\/images?q=tbn:ANd9GcT022992NVXr0X_4MxgiyAiDNaAEEeLQWQG13uLzmmHhvzB-qSyalMCGW2vcYo&s",
        "https:\/\/encrypted-tbn0.gstatic.com\/images?q=tbn:ANd9GcQd2tO_-bvzH6JW7tmrd3G4V3lzUBdES6NNrSRwkWrJiC5WNpq2CdyH70bdly8&s",
        "https:\/\/encrypted-tbn0.gstatic.com\/images?q=tbn:ANd9GcQRM66XSkowxpcaGOfiVWR3ybi6yHFq_Sh5U2nTpIIPItKKyswkjfli2KszdQ&s",
        "https:\/\/encrypted-tbn0.gstatic.com\/images?q=tbn:ANd9GcTN2cQhlNT32jKrvbTxy0559fQVQ952ch8mLTrK-iKs0z6OKXBC0xsYZcC-JA&s",
        "https:\/\/encrypted-tbn0.gstatic.com\/images?q=tbn:ANd9GcQ6WoWX7owbiY6V3y6DwCHbslI6_e4AXbX_auYDGtS22UpxGO1N9EIUEIHs_pI&s",
        "https:\/\/encrypted-tbn0.gstatic.com\/images?q=tbn:ANd9GcSbzzC3YheBAaM848jhszxGdgJ8T51ktov7AkF1_o3yOh-ogFDhuAKoWyvj3vo&s",
        "https:\/\/encrypted-tbn0.gstatic.com\/images?q=tbn:ANd9GcRpbwqR7yp7po70VgFoPLvTu9ttRjv1EFFUplLga0LFme0CXx_fm22C3FsmmYc&s",
        "https:\/\/encrypted-tbn0.gstatic.com\/images?q=tbn:ANd9GcQWEfrnCMHN6MpwIwIALzdfxFCSIABDE3ujNzEtK6vd9cGh1mpGRrVw9A4mlw&s"
    ]
}
```

maybe some proxy issue?

**Comment by suzizisu** [comment]
As you tested, I opened a new machine on vultr to test, and the search can work normally. My local network environment is the same. My guess is that the system is impure or something is affecting the search. For example wordpress or php?

**Comment by STillSB** [comment]
> As you tested, I opened a new machine on vultr to test, and the search can work normally. My local network environment is the same. My guess is that the system is impure or something is affecting the search. For example wordpress or php?

Are you in China? It must caused by GFW

**Comment by ellite** [comment]
Might be fixed by 1.16.3

**Comment by suzizisu** [comment]
After testing, the search function is now working properly.Thanks
